### PR TITLE
Edx global analytics ginkgo rg

### PR DESCRIPTION
**Pull request by KhaimovMR**
Global Analytics (including the updates from the [OLGA-24](https://youtrack.raccoongang.com/issue/OLGA-24))

### CODE REVIEW

File: lms/urls.py [hunk 1]
```diff
@@ -1034,3 +1034,15 @@
         )
     )
 
+# Include edX global analytics application urls.
```
Comment (idegtiarov) [inline lms/urls.py]: It is redundant change

File: openedx/core/djangoapps/edx_global_analytics/tasks.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    """
+    Set last sent dates to the cache.
+
+    :param result: result of the http client request that were sent a statistics to the OLGA
+    :param statistics_level: statistics level - may be "paranoid" or "enthusiast"
+    :param token: access token used to push statistics and to distinguish the cache elements
+    :param type: type of the data ("registered", "certificates" or "enthusiastic")
+    :param data: data as a dictionary where key is a date and the value is a count of elements at this day
+    """
+    if result != True:  # If http request is failed
```
Comment (idegtiarov) [inline openedx/core/djangoapps/edx_global_analytics/tasks.py]: I would use `not result` here

**Comment by hetmantsev** [comment]
@KhaimovMR сейчас поправлю ci. ошибка вылезла.

**Comment by hetmantsev** [comment]
Jenkins, test this please.

**Comment by hetmantsev** [comment]
Jenkins, test this please.

**Pull request closed without merging**